src/java.base/share/classes/sun/launcher/LauncherHelper.java

```diff
@@ -408,6 +408,11 @@ public static void printSystemMetrics() {
         limit = c.getPidsMax();
         ostream.println(formatLimitString(limit, INDENT + "Maximum Processes Limit: ",
                                           longRetvalNotSupported, false));
+
+        long val = c.getPidsCurrent();
+        ostream.println(formatLimitString(val, INDENT + "Current number of processes: ",
+                                          longRetvalNotSupported, false));
+
```

Comment:
I'm not sure we should add this to `-XshowSettings:system` output at all. What's reported there is enforced limits. Note that current memory usage isn't shown either.